src/SFA.DAS.PR.Jobs/Infrastructure/IRoatpServiceApiClient.cs

```diff
@@ -5,6 +5,6 @@ namespace SFA.DAS.PR.Jobs.Infrastructure;
 
 public interface IRoatpServiceApiClient
 {
-    [Get("/api/v1/fat-data-export")]
+    [Post("/api/v1/fat-data-export")]
```

Comment:
Why was this changed to POST it should be GET

Reply:
@DevenKShah Not sure this was intentional, i'll revert it.